app/src/components/pages/collectionLanePage/collectionLanePage.tsx

```diff
@@ -10,17 +10,17 @@ import PageLayout from "../../pageLayout/pageLayout";
 import { useParams } from "next/navigation";
 import { headerBreakpoints } from "../../../utils/breakpoints";
 import { slugToString } from "../../../utils/utils";
-import CollectionCard from "../../../components/cards/collectionCard";
```

Comment:
There are other imports you can remove since they are not being used:
`SimpleGrid`, `PageLayout`, `DCCard`, `CollectionCardModel`, `useBreakpoints`, `CollectionDataType`, and `DCSimpleGrid`. Perhaps slightly out of scope of this PR but pointing it out since `DCCard` and `CollectionsGrid` are imported.